sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamBuiltinAggregations.java

```diff
@@ -397,6 +399,93 @@ public Long extractOutput(Long accum) {
    * <p>Note: null values are ignored when mixed with non-null values.
    * (https://issues.apache.org/jira/browse/BEAM-10379)
    */
+  //  static class BitAnd<T extends Number> extends CombineFn<T, Long, Long> {
```

Comment:
This PR and https://github.com/apache/beam/pull/13391 both have change to add back the already deleted lines. I think this is unintended? Could you please remove these from these 2 PRs?